### PR TITLE
sl docs fast follows based on user feedback

### PR DESCRIPTION
**Pull request by Jstein77**
SL fast follows to address user feedback captured in [this thread](https://dbt-labs.slack.com/archives/C0484Q63MU5/p1691345353142199?thread_ts=1690576552.157689&cid=C0484Q63MU5)

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ dbt Labs partners can use the JDBC API to build integrations in their tools with
 
 If you are a dbt user or partner with access to dbt Cloud and the[dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), you can [setup](/docs/use-dbt-semantic-layer/setup-sl) and test this API with data from your own instance by configuring the Semantic Layer and obtaining the right JDBC connection parameters described in this document.
 
-Refer to [Get started with the dbt Semantic Layer](docs/use-dbt-semantic-layer/quickstart-sl) for more info.
+Refer to [Get started with the dbt Semantic Layer](../use-dbt-semantic-layer/quickstart-sl) for more info.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
Refer to [Get started with the dbt Semantic Layer](/docs/use-dbt-semantic-layer/quickstart-sl) for more info.
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ dbt Labs partners can use the JDBC API to build integrations in their tools with
 
 If you are a dbt user or partner with access to dbt Cloud and the[dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), you can [setup](/docs/use-dbt-semantic-layer/setup-sl) and test this API with data from your own instance by configuring the Semantic Layer and obtaining the right JDBC connection parameters described in this document.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
If you are a dbt user or partner with access to dbt Cloud and the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), you can [setup](/docs/use-dbt-semantic-layer/setup-sl) and test this API with data from your own instance by configuring the Semantic Layer and obtaining the right JDBC connection parameters described in this document.
```

File: website/docs/docs/build/metrics-overview.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ The keys for metrics definitions are:
 | Component | Description | Type |
 | --------- | ----------- | ---- |
 | `name` | Provide the reference name for the metric. This name must be unique amongst all metrics.   | Required |
-| `type` | Define the type of metric, which can be a measure (`simple`) or ratio (`ratio`)).  | Required |
+| `type` | Define the type of metric, which can be `simple`, `ratio`, `cumulative` or `derived`.  | Required |
```
Comment (mirnawong1) [inline website/docs/docs/build/metrics-overview.md]:
```suggestion
| `type` | Define the type of metric, which can be `simple`, `ratio`, `cumulative`, or `derived`.  | Required |
```

File: website/docs/docs/build/cumulative-metrics.md [hunk 4]
```diff
@@ -19,9 +19,8 @@ metrics:
     label: The value that will be displayed in downstream tools # Required
     type_params: # Required
       measure: the measure you are referencing # Required
-      window: the accumulation window i.e 1 month, 7 days, 1 year. # Optional. Cannot be used with grain_to_date
-      grain_to_date: sets the accumulation grain, such as month will accumulate data for one month, then restart at the beginning of the next. 
-      # Optional. Can not be used with window. 
+      window: the accumulation window i.e 1 month, 7 days, 1 year. # Optional. Can not be used with window.
```
Comment (mirnawong1) [inline website/docs/docs/build/cumulative-metrics.md]: @Jstein77 this example has `type: simple`. why isn't it `type: cumulative`?

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
lgtm! ready to merge!

File: website/docs/docs/build/cumulative-metrics.md (hunk 4 above)
Comment (Jstein77) [reply website/docs/docs/build/cumulative-metrics.md]: Yea it should be cumulative.

Comment (mirnawong1) [reply website/docs/docs/build/cumulative-metrics.md]: thanks @Jstein77 , will update this in the other fast follow pr

**Pull request merged**